packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -40,7 +40,7 @@ class RenderProxyBox extends RenderBox with RenderObjectWithChildMixin<RenderBox
   /// Proxy render boxes are rarely created directly because they simply proxy
   /// the render box protocol to [child]. Instead, consider using one of the
   /// subclasses.
-  RenderProxyBox([RenderBox child = null]) {
+  RenderProxyBox([RenderBox child = null]) { //ignore: avoid_init_to_null
```

Comment:
@bwilkerson : there shouldn't be analyzer skew since we're pulling in the analyzer from the SDK via a dependency_override:


```
dependency_overrides:
  analyzer:
    path: ../../bin/cache/dart-sdk/lib/analyzer
```